shared/rollup.config.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+import { readFileSync } from "fs";
+import { join } from "path";
+import { cwd } from "process";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import terser from "@rollup/plugin-terser";
+
+/**
+ * Create a base rollup config
+ *
+ * @param {object} [options] Configuration object
+ * @param {string} [options.input] Input path
+ * @param {import('rollup').ExternalOption} [options.external] External dependencies list
+ * @param {import('rollup').RollupOptions | import('rollup').RollupOptions[]} [options.additionalConfigs] Additional rollup configurations
+ *
+ * @returns {import('rollup').RollupOptions}
+ */
+export function createConfig(options = {}) {
+  const {
+    input = "guest-js/index.ts",
+    external = [/^@tauri-apps\/api/],
+    additionalConfigs = [],
+  } = options;
+
+  const pkg = JSON.parse(readFileSync(join(cwd(), "package.json"), "utf8"));
+
+  const pluginJsName = pkg.name
+    .replace("@tauri-apps/plugin-", "")
+    .replace(/-./g, (x) => x[1].toUpperCase());
+  const iifeVarName = `__TAURI_PLUGIN_${pluginJsName.toUpperCase()}__`;
```

Comment:
Shouldn't `-` be converted to `_` for the global vars?

Reply:
I don't know about this logic, it is the same as the old one, I only added `_PLUGIN_`

Reply:
We don't need to preserve the separator between `global` and `shortcut` since it is a temporary variable and ultimately is assigned to `window.__TAURI__.globalShortcut`